### ISSUE TITLE
Bug; refetchResourcesOnNavigate:true passing start & end does not work from refetchResources

### ISSUE THREAD

**Issue by JamesValleyT** [fullcalendar/fullcalendar-scheduler#296]
Go to https://jsbin.com/bivomab/1/edit?html,output

Open Developer Console

Click "Run With JS"

Note that start and end, as written to the console, has values - as it should.

Now click the button "CLICK ME"

Note start and end, as written to the console, are Null. This is the bug - shouldn't there be values?

**Comment by arshaw** [comment]
Recreated. thanks

**Comment by margusarm** [comment]
Just noticed this too.

Might not be useful info but when using the old solution by arshaw, start&end work with refetchResources:
```
	resources:
	function (callback) {
		setTimeout(function () {
			var view = $('#calendar').fullCalendar('getView');
			$.ajax({
				url: 'json_resources.php',
				dataType: 'json',
				data: {
					start: view.start.format(),
					end: view.end.format(),
					timezone: view.options.timezone,
				}

			}).then(function (resources) {
				callback(resources)
			});

		}, 0);
	},
```

**Comment by arshaw** [comment]
fixed and queued up for the next release
https://github.com/fullcalendar/fullcalendar-scheduler/blob/master/tests/refetchResourcesOnNavigate.coffee#L229

**Comment by JamesValleyT** [comment]
Thanks! :-)

**Comment by arshaw** [comment]
Released in [1.6.2](https://github.com/fullcalendar/fullcalendar-scheduler/releases/tag/v1.6.2).
